test/LibraryManager.Test/LibrariesValidatorTest.cs

```diff
@@ -33,43 +34,77 @@ public void Setup()
         }
 
         [TestMethod]
-        public void DetectConlictsAsync_ConflictingFiles()
+        public async Task DetectConlictsAsync_ConflictingFiles()
         {
+            string expectedErrorCode = "LIB016";
+            string expectedErrorMessage = "Conflicting file \"lib\\jquery.js\" found in more than one library: jquery@3.1.1, jquery@2.2.1";
             _manifest = Manifest.FromJson(_docConflictingLibraries, _dependencies);
```

Comment:
>_manifest [](start = 12, length = 9)

_manifest is initialized in every test, it should be a local instead of class field